packages/launchpad/src/components/PackagesList.vue

```diff
@@ -26,17 +26,9 @@ fragment PackagesList on Wizard {
 }
 `
 
-export default defineComponent({
-  props: {
-    gql: {
-      type: Object as PropType<PackagesListFragment>,
-      required: true
-    }
-  },
-  setup(props) {
-    return { 
-      packagesToInstall: computed(() => props.gql.packagesToInstall)
-    };
-  },
-});
+const props = defineProps<{
+  gql: PackagesListFragment
+}>()
+
+const packagesToInstall = computed(() => props.gql.packagesToInstall)
```

Comment:
Typing `props.` gives us immediate TS inference - no awkward pause while the Volar language server figures out what to do.